src/components/AssetListItem.tsx

```diff
@@ -0,0 +1,31 @@
+import { useId } from 'react';
+import { useAtomValue } from 'jotai';
+
+import { displayFunctionsAtom } from 'store/app';
+import type { Brand } from 'store/app';
+
+const AssetListItem = ({ brand }: Brand) => {
```

Comment:
This is saying that the object with `brand` inside has type `Brand`, which can't be right. The reason TS isn't complaining is that `Brand` is typed `any`. I know it's a bit of a pain copying over the types from agoric-sdk but it's worth it to get static type checking, autocomplete, etc.

I've pushed a commit with the necessary ERTP types. If there are a lot more we can do a one-off to build its `.d.ts`.